### PR TITLE
Klevu Documentation

### PR DESCRIPTION
**Pull request by rahinisrinivas**

### CODE REVIEW

File: docs/composer/integration-hub/site-search/klevu.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
+---
+title: Integrating with Klevu Catalog Sync
+nav_label: Klevu Catalog Sync
+sidebar_position: 41
+---
+
+You can integrate Commerce with Klevu quickly and easily in Commerce Manager. Klevu is an advanced AI-driven search and product discovery platform specifically designed for e-commerce websites. Once you have integrated Klevu, you can synchronize catalog from Commerce Manager to Klevu.
+
```
Comment (johughes1204) [inline docs/composer/integration-hub/site-search/klevu.md:8]:
```suggestion
The integration is responsible for server-side tracking, for example, Klevu events. Your business also requires a frontend component. For example, when customers view product detail pages or search pages, you can integrate Klevu in your frontend to track these events. See [Integrating Klevu in Studio](/docs/studio/Integrations/klevu/getting-started) for an example of using Klevu in a storefront.
```

I am not 100% sure about this text, so maybe ask Scott, but we added something like this to the segment integration so I wondered if we needed to add something about studio here as well.

File: docs/composer/integration-hub/site-search/klevu.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+1. In Commerce Manager, go to **COMPOSER** > **Integrations Hub**.
+1. Under **Search** click on the **Klevu Catalog Sync** Integration.
+1. Click **Configure**. The **Trigger details** are displayed.
+1. Click **Next**. The **Connections** page is displayed.
+1. Enter the **Elastic Path Commerce Cloud Connection** details. See [Collecting Commerce Manager Setup Information](#collecting-commerce-manager-setup-information).
+1. Click **Connect**. A new window is displayed to confirm the connection was successful. If successful, close this window and then click **Next**.
+1. By default, **Fetch Main Image URL** and **Fetch Inventory** toggles are enabled. You can modify the default configuration policy, batch size and concurrency settings if necessary. We recommend keeping these settings as default.
+1. Click **Next**.
+1. Provide the **Base Product URL** and **Base Category URL** relevant to your products and categories.
```
Comment (johughes1204) [inline docs/composer/integration-hub/site-search/klevu.md:32]: would a customer know what these are? Do we need to provide an example ?

File: docs/composer/integration-hub/site-search/klevu.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+1. Click **Finish** to complete the configuration process.
+
+## Understanding Klevu Mappings
+
+Once you have configured the integration, it creates:
+
+- A Custom API named **Klevu Keys** under **COMMERCE EXTENSIONS** to map Klevu store credentials for publishing the catalog.
+- A flow named **Klevu Mapping** under **SYSTEM > Flows** for mapping Klevu and custom field.
```
Comment (johughes1204) [inline docs/composer/integration-hub/site-search/klevu.md:40]:
```suggestion
- A flow named **Klevu Mapping** under **SYSTEM > Flows** for mapping Klevu and custom fields. 
```
Is there more than one?

File: docs/composer/integration-hub/site-search/klevu.md [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+
+## Understanding Klevu Mappings
+
+Once you have configured the integration, it creates:
+
+- A Custom API named **Klevu Keys** under **COMMERCE EXTENSIONS** to map Klevu store credentials for publishing the catalog.
+- A flow named **Klevu Mapping** under **SYSTEM > Flows** for mapping Klevu and custom field. 
+- A Webhook event named **Klevu Catalog Sync**.
+
+The following table describes fields of the Commerce Product Fields and their corresponding Klevu attributes:
```
Comment (johughes1204) [inline docs/composer/integration-hub/site-search/klevu.md:43]:
```suggestion
The following table describes the Commerce product fields and their corresponding Klevu attributes:
```

**Review by johughes1204 — APPROVED** [review]

File: docs/composer/integration-hub/site-search/klevu.md (hunk 1 above)
Comment (rahinisrinivas) [reply docs/composer/integration-hub/site-search/klevu.md:8]: Confirmed with Scott that this is not required.

**Pull request merged**